### PR TITLE
Intro Added to Commits/Notification Category

### LINKED ISSUE #6794: Intro for notifications category
### What article on docs.github.com is affected?

The "[Managing subscriptions and notifications on GitHub](https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github)" category of articles.

### What part(s) of the article would you like to see updated?

Write intros to help orient users to the content in the category. Intros should be one sentence long. For more information on writing intros, see:
- [Information about categories](https://github.com/github/docs/blob/main/contributing/content-model.md#category) in the GitHub Docs content model
- [Information about writing intros](https://github.com/github/docs/blob/main/contributing/content-model.md#intro) in the GitHub docs content model
- [Information about the `intro` frontmatter property](https://github.com/github/docs/tree/main/content#intro) in the GitHub Docs content directory README
- Make sure your intro is general enough to cover the content in each version of the page (use the version picker in the upper-right corner of the page to see what’s included on the page for different versions of GitHub)

### Additional information

This issue is for the Campus Experts + Docs program and is not available for contributors outside the program to work on.

### LINKED ISSUE #6795: Introduction for commits category
### What article on docs.github.com is affected?

The "[Committing changes to your project](https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github)" category of articles.

### What part(s) of the article would you like to see updated?

Write intros to help orient users to the content in the category. Intros should be one sentence long. For more information on writing intros, see:
- [Information about categories](https://github.com/github/docs/blob/main/contributing/content-model.md#category) in the GitHub Docs content model
- [Information about writing intros](https://github.com/github/docs/blob/main/contributing/content-model.md#intro) in the GitHub docs content model
- [Information about the `intro` frontmatter property](https://github.com/github/docs/tree/main/content#intro) in the GitHub Docs content directory README
- Make sure your intro is general enough to cover the content in each version of the page (use the version picker in the upper-right corner of the page to see what’s included on the page for different versions of GitHub)

### Additional information

This issue is for the Campus Experts + Docs program and is not available for contributors outside the program to work on.

### PR DESCRIPTION
**Pull request by codewithdev**
### Why:

Closes #6794 #6795 

### What's being changed:

- Intro added for [Managing Subscriptions and Notifications on GitHub](https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github)
- Intro Added for [Committing changes to your project](https://docs.github.com/en/github/managing-subscriptions-and-notifications-on-github)

### Check off the following:

- [x] I have reviewed my changes in staging (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Review by Ely68 — APPROVED** [review]

**Review by Ely68 — APPROVED** [review]

**Review by Ely68 — APPROVED** [review]

**Review by Ely68 — APPROVED** [review]

**Review by Ely68 — APPROVED** [review]

**Review by Ely68 — APPROVED** [review]

**Review by Ely68 — APPROVED** [review]

**Comment by janiceilene** [comment]
@codewithdev Thanks so much for opening a PR for this intro 🙌 I'll get this triaged for review ⚡

**Review by mchammer01 — COMMENTED** [review]
@codewithdev - Introductions are not easy and you've made a great job ✨ 
I've left you a few pointers to improve the intro in an inline comment.
Hope this helps!

**Review by Ruokrscln — DISMISSED** [review]

**Review by Chucky-ux — APPROVED** [review]

File: content/github/managing-subscriptions-and-notifications-on-github/index.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Managing subscriptions and notifications on GitHub
+intro: 'To get the updates about your workflows and activities on {% data variables.product.company_short %} ,you can configure and manage all your subscriptions.'
```
Comment (felicitymay) [inline content/github/managing-subscriptions-and-notifications-on-github/index.md:3]: Again, I'd recommend that we avoid the term "workflows". I suggest:

```suggestion
intro: 'You can specify how to receive notifications, the repositories you are interested in, and the types of activity you want to hear about.'
```

**Review by felicitymay — COMMENTED** [review]
Thanks for adding introductions. I've suggested updates to reduce the potential for confusion and to make them a little broader.

**Review by felicitymay — APPROVED** [review]

**Comment by codewithdev** [comment]
> Thanks for adding introductions. I've suggested updates to reduce the potential for confusion and to make them a little broader.

@felicitymay Thanks for adding the suggestion. I will improve and make sure to add keywords that sound simple and convenient from the user's perspective.

**Pull request merged**